fastqheat/__main__.py

```diff
@@ -6,18 +6,31 @@
 import ssl
 import subprocess
 from pathlib import Path
+from typing import Any
 
+import backoff
 import requests
 import urllib3
 
 from fastqheat.check import check_loaded_run, md5_checksum
+from fastqheat.config import get_settings, set_settings
 from fastqheat.metadata import ENAClient
 
 SRR_PATTERN = re.compile(r'^(SRR|ERR|DRR)\d+$')
 SRP_PATTERN = re.compile(r'^(((SR|ER|DR)[PAXS])|(SAM(N|EA|D))|PRJ(NA|EB|DB)|(GS[EM]))\d+$')
 USABLE_CPUS_COUNT = len(os.sched_getaffinity(0))
 
 
+@backoff.on_exception(
+    backoff.constant,
+    subprocess.CalledProcessError,
+    max_tries=lambda: get_settings().max_retries,
+)
+def _run_command(args: Any) -> None:
```

Comment:
There is no need in defining a separate function. You can decorate `subprocess.run()` itself:
```python
subprocess_run = backoff.on_exception(blabla)(subprocess.run)
```

Reply:
If you think that this looks better -- sure.